pkg/apis/networking/types.go

```diff
@@ -138,10 +138,19 @@ type NetworkPolicyPort struct {
 	// +optional
 	Protocol *api.Protocol
 
-	// The port on the given protocol. This can either be a numerical or named port on
-	// a pod. If this field is not provided, this matches all port names and numbers.
+	// The port on the given protocol. This can either be a numerical or named
+	// port on a pod. If this field is not provided, this matches all port names and
+	// numbers.
+	// If present, only traffic on the specified protocol AND port will be matched.
 	// +optional
 	Port *intstr.IntOrString
+
+	// endPort, if set, indicates that a range of ports, from port to endPort inclusive
+	// should be allowed by the policy. This field cannot be defined if the port field
+	// is not defined or if the port field is defined as a named (string) port.
+	// Also the endPort must be equal or greater than port
+	// +optional
+	EndPort *int32
```

Comment:
IntOrString defines the Port's integer field as `int`. Any particular reason to use `int32` for EndPort ?

Reply:
we don't use plain `int` in api fields. The fact that `IntOrString` is internally an `int` is probably a historical error

Reply:
thanks.. good to know

Reply:
I guess this is resolved, right?

Reply:
yes.. this is resolved :)